versioned_docs/version-v1.28.x/getting-started/release-notes/v1_28_5.md

```diff
@@ -39,27 +18,54 @@ Zowe Version 1.28.5 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
+- Updated Imperative to version `4.18.19` to fix issues with normalizing newlines on file uploads. ([#1815](https://github.com/zowe/zowe-cli/issues/1815))
+
+- Updated Imperative to version `4.18.18` to use `@zowe/secrets-for-zowe-sdk` as the default credential manager.
```

Comment:
@adam-wolfe does this need a PR/issue link?

Reply:
Not sure. Also, we updated imperative to 4.18.17 then to 4.18.18 and 4.18.19. Does it make sense to list all of those updates if the end result is that Zowe CLI for the 1.28.5 release is using v4.18.19 of imperative? Might be a good thing to discuss with the squad.

Reply:
Done -- thank you!